src/components/reusables/SummaryPage/SummaryPage.tsx

```diff
@@ -1,22 +1,22 @@
-import { Panel, PanelVariant } from '@altinn/altinn-design-system';
+import {
+  Page,
+  PageContent,
+  PageHeader,
```

Comment:
Are we going back to the page from the design system or is this a case of git acting up? 😅

Reply:
git acting up